docs/machine-learning/tutorials/image-classification-api-transfer-learning.md

```diff
@@ -0,0 +1,451 @@
+
+## ML.NET Image Classification API
+
+ML.NET provides various ways of performing image classification. This tutorial focuses on the Image Classification API. The Image Classification API makes use of [TensorFlow.NET](https://github.com/SciSharp/TensorFlow.NET), a low-level library that provides C# bindings for the TensorFlow C++ API.
+
+![ML.NET Image Classification API Architecture](./media/image-classification-api-transfer-learning/architecture.png)
+
+### What is transfer learning?
+
+Transfer learning is the process of using knowledge gained while solving one problem and applying it to a different but related problem.
+
+Training a deep learning model from scratch requires setting several parameters, a large amount of labeled training data and a vast amount of compute resources (hundreds of GPU hours). Using a pre-trained model along with transfer learning allows you to shortcut the training process. 
+
+### Training process
+
+The Image Classification API starts the training process by loading a pre-trained TensorFlow model. The training process consists of two steps:
+
+1. Bottleneck phase
+2. Training phase
+
+![Training Steps](./media/image-classification-api-transfer-learning/training.png)
+
+#### Bottleneck phase
+
+During the bottleneck phase, the set of training images are loaded and the pixel values are used as as input, or features, for the frozen layers of the pre-trained model. The frozen layers include all of the layers in the neural network up to the penultimate layer, informally known as the bottleneck layer. These layers are referred to as frozen because no training will occur on these layers and operations are pass-through. It's at these frozen layers that the lower-level patterns that help a model differentiate between the different classes are computed. The larger the number of layers, the more computationally intensive this step is. Fortunately, since this is a one-time calculation, the results can be cached and used in later runs when experimenting with different parameters.
+
+#### Training phase
+
+Once the output values from the bottleneck phase are computed, they are used as input to retrain the final layer of the model. This process is iterative and runs for the number of times specified by model parameters. During each run, the loss and accuracy are evaluated and the appropriate adjustments are made to improve the model with the goal of minimizing the loss and maximizing the accuracy. Once training is finished, two model formats are output. One of them is the `.pb` version of the model and the other is the `.zip` ML.NET serialized version of the model. When working in environemnts supported by ML.NET, it is recommended to use the `.zip` version of the model. However, in environemnts where ML.NET is not supported, you have the option of using the `.pb` version.
```

Comment:
@cjgronlund it's easier to submit corrections via suggestions and in bulk to keep our inboxes sane. Did an example below.

![image](https://user-images.githubusercontent.com/12971179/67248541-ab7d8300-f419-11e9-867f-f4b2b11c5b4b.png)


```suggestion
Once the output values from the bottleneck phase are computed, they are used as input to retrain the final layer of the model. This process is iterative and runs for the number of times specified by model parameters. During each run, the loss and accuracy are evaluated and the appropriate adjustments are made to improve the model with the goal of minimizing the loss and maximizing the accuracy. Once training is finished, two model formats are output. One of them is the `.pb` version of the model and the other is the `.zip` ML.NET serialized version of the model. When working in environments supported by ML.NET, it is recommended to use the `.zip` version of the model. However, in environments where ML.NET is not supported, you have the option of using the `.pb` version.
```

Reply:
Thanks for the tip. Sorry for the spam!

Reply:
No worries. Suggestions is an awesome feature and then we can later apply them in bulk as well.